angular-project/src/app/app.module.ts

```diff
@@ -6,19 +6,24 @@ import { NgModule } from '@angular/core';
 import { AppComponent } from './app.component';
 import { AppRoutingModule } from './app-routing.module';
 import { NavBarComponent } from './navbar/navbar.component';
+import { HomeComponent } from './home/home.component';
+import { AuthHttp, AUTH_PROVIDERS, provideAuth, AuthConfig } from  'angular2-jwt/angular2-jwt';
```

Comment:
replace this with httpClientModule
https://angular.io/guide/http

Reply:
Will resolve this issue on the next commit. Merging on master now.